### PR TITLE
Fixes #35370 - Add ouia-ids to CV UI components

### PR DESCRIPTION
**Pull request by sjha4**
#### What are the changes introduced in this pull request?
Add ouiaIds to all supported components in CV UI

#### What are the testing steps for this pull request?
Follow instructions in comments in katello/webpack/global_test_setup.js
In katello dir, Run `npx jest webpack/scenes/ContentViews/`

### CODE REVIEW

**Comment by jeremylenz** [comment]
[test katello]

File: webpack/global_test_setup.js [hunk 1]
```diff
@@ -6,9 +6,28 @@ import nock from 'nock';
...
+  // in case of ouia-id check
+
+  /* Uncomment block below to filter out PF4 ouiaId errors */
+  /*
+  const ignore = ['in ForwardRef (created by Tabs)',
+     'in Button (created by Label)', 'created by _default',
+     'created by PaginationOptionsMenu', 'created by TypeAheadItems',
+     'in Modal (created by Wizard)',
+     'created by WizardHeader', 'created by Navigation',
+     'The prop `ouiaId` is marked as required in `RowWrapper`',
```
Comment (jeremylenz) [inline webpack/global_test_setup.js:19]: RowWrapper is specifically listed as a supported component, so it's on the list in ouia_check.js. If it's not useful to have it for some reason, should probably just delete it there.

**Review by jeremylenz — COMMENTED** [review]
Thanks @sjha4!

Looks like a few were missed.. (I ran it without the ignores.)

TableComposable in 

```
/home/jeremylenz/code/centos8-katello-devel-stable/katello/webpack/scenes/ContentViews/Delete/Steps/CVDeleteEnvironmentsSelection.js
  67,17:                 <TableComposable variant={TableVariant.compact}>

/home/jeremylenz/code/centos8-katello-devel-stable/katello/webpack/scenes/ContentViews/Details/Versions/BulkDelete/Steps/ReviewEnvironments.js
  67,7:       <TableComposable variant={TableVariant.compact}>

/home/jeremylenz/code/centos8-katello-devel-stable/katello/webpack/scenes/ContentViews/Details/Versions/Delete/RemoveSteps/CVEnvironmentSelectionForm.js
  79,9:         <TableComposable variant={TableVariant.compact}>

/home/jeremylenz/code/centos8-katello-devel-stable/katello/webpack/scenes/ContentViews/Publish/CVPublishReview.js
  45,7:       <TableComposable aria-label="Review Table">
```

and `<Tr>` in several files..

I think the ignores were a bit too enthusiastic..

File: webpack/global_test_setup.js (hunk 1 above)
Comment (sjha4) [reply webpack/global_test_setup.js:19]: We don't use RowWrapper anywhere..yet..It's called by TableComposable or some other component we use. I don't know if we'll start using it anytime soon..Can do it either way.

Comment (jeremylenz) [reply webpack/global_test_setup.js:19]: I'd comment it out in ouia_check.js then

**Comment by sjha4** [comment]
All those files probably need a refactor to use Composable table. These seem to be on the old style tables with an array of cells passed for a row instead of the TableComposable version.

**Comment by jeremylenz** [comment]
> All those files probably need a refactor to use Composable table. These seem to be on the old style tables with an array of cells passed for a row instead of the TableComposable version.

Not sure I'm following..? Those files use `<TableComposable>` already; they just need `ouiaId` props..

**Comment by sjha4** [comment]
My bad..Those can have ouia-ids added. Not sure why I wasn't seeing this error after removing ignores. Will update..

**Comment by sjha4** [comment]
@jeremylenz : Updated to cover modals that I had happily ignored in the list..

**Review by jeremylenz — APPROVED** [review]
Thanks @sjha4 

let's make QE happy!

ACK :+1:

**Pull request merged**